plutus-core/plutus-core/src/PlutusCore/Annotation.hs

```diff
@@ -98,8 +99,10 @@ instance Show SrcSpan where
 instance Pretty SrcSpan where
     pretty = viaShow
 
+instance NFData SrcSpan
+
 newtype SrcSpans = SrcSpans {unSrcSpans :: Set SrcSpan}
-    deriving newtype (Eq, Ord, Semigroup, Monoid, MonoFoldable)
+    deriving newtype (Eq, Ord, Semigroup, Monoid, MonoFoldable, NFData)
```

Comment:
I'm actually not sure whether it's correct to newtype derive `NFData`? I guess it must be since it's representationally equal to the underlying type.

Reply:
Yes, I think so, and I've never heard of anyone recommending against doing that.